### PR TITLE
Buttons behaviours and colours update

### PR DESCRIPTION
**Pull request by blazeque**
| - | - |
| --- | --- |
| Decisions |  Buttons behaviours and colours update |
| Animated GIF | dadada |

### CODE REVIEW

File: vue/components/ui/button-color-platforme/button-color-platforme.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@
     cursor: pointer;
     display: inline-block;
     font-size: 13px;
-    font-weight: 600;
```
Comment (gcandal) [inline vue/components/ui/button-color-platforme/button-color-platforme.vue]: Let's leave this as it was, even if it's different from Zeplin

File: vue/components/ui/button-color-platforme/button-color-platforme.vue [hunk 2]
```diff
@@ -40,48 +40,167 @@
     min-width: 160px;
 }
 
-.button-color.disabled {
-    cursor: default;
-    opacity: 0.4;
+.button-color.button-color-default.disabled {
+    background-color: #eceef1;
+    border: 2px solid #eceef1;
```
Comment (gcandal) [inline vue/components/ui/button-color-platforme/button-color-platforme.vue]: If the border width (2px) and style (solid) is the same across all colors/states, then define it at the `button-color` level.

Then, you just have to change the `border-color` property on each state.

File: vue/components/ui/button-color-platforme/button-color-platforme.vue [hunk 3]
```diff
@@ -40,48 +40,167 @@
...
+.button-color.button-color-red:active {
+    background-color: #ce544d;
+    border: 2px solid #ce544d;
+}
+
+.button-color.button-color-red.disabled {
+    background-color: #ffe4e2;
+    border: 2px solid #ffe4e2;
+    color: #a4adb5;
+    cursor: not-allowed;
```
Comment (gcandal) [inline vue/components/ui/button-color-platforme/button-color-platforme.vue]: The only thing color selectors should change are, well, colors. Move this cursor thing to a `.button-color.disabled` and delete from everywhere else.

**Comment by gcandal** [comment]
On top of all the changes, place the colors in the `variables.css` file. Example: https://github.com/ripe-tech/ripe-components-vue/pull/46#issuecomment-559056988

File: vue/components/ui/input-platforme/input-platforme.vue [hunk 4]
```diff
@@ -19,7 +19,6 @@ input[type="text"] {
     border: 1px solid transparent;
     border-radius: 6px 6px 6px 6px;
     box-sizing: border-box;
-    font-family: $font-family;
```
Comment (gcandal) [inline vue/components/ui/input-platforme/input-platforme.vue]: revert

File: vue/components/ui/button-color-platforme/button-color-platforme.vue (hunk 1 above)
Comment (blazeque) [reply vue/components/ui/button-color-platforme/button-color-platforme.vue]: Now, all the button use font-weight: bold. I defined globally

**Comment by joamag** [comment]
@gcandal @jorgefernandess we need to revisit the color @gcandal ping me about this

**Comment by gcandal** [comment]
Too many conflicts here, opening a new one based on this: https://github.com/ripe-tech/ripe-components-vue/pull/148

**Pull request closed without merging**